petric.py

```diff
@@ -110,7 +110,7 @@ def __call__(self, algo: Algorithm):
 
 class QualityMetrics(ImageQualityCallback, Callback):
     """From https://github.com/SyneRBI/PETRIC/wiki#metrics-and-thresholds"""
-    def __init__(self, reference_image, whole_object_mask, background_mask, interval: int = 1 << 31, **kwargs):
+    def __init__(self, reference_image, whole_object_mask, background_mask, interval: int = 1, **kwargs):
```

Comment:
```suggestion
    def __init__(self, reference_image, whole_object_mask, background_mask, interval: int = 3, **kwargs):
```
Setting it to 3 to avoid too many computations for now. Will need to be set to 1 for evaluation